resolver_service/config.py

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
```

Comment:
i have removed most of the contents here for these reasons:

- LOGGING definition will be part of adsmutils
- SQL strings were converted to SQLALCHEMY ORM
- error messages are better to be written into the code (in general, avoid too much of meta statements; simple trumps 'flexible' in this case)